httpie/client.py

```diff
@@ -104,10 +104,15 @@ def collect_messages(
                     **send_kwargs,
                 )
 
-            # noinspection PyProtectedMember
-            expired_cookies += get_expired_cookies(
-                headers=response.raw._original_response.msg._headers
-            )
+            if isinstance(response.raw, urllib3.HTTPResponse):
+                # noinspection PyProtectedMember
+                expired_cookies += get_expired_cookies(
+                    headers=response.raw._original_response.msg._headers
```

Comment:
Its not entirely clear to me why `response.raw._original_response.msg._headers` is being utilized here instead of `response.headers`. I did run tests without making use of  `response.raw._original_response.msg._headers` and they passed. Logic is left in place to ensure compatibility for what ever case this was intended for, however would be happy to remove on recommendation from someone more experienced with the project.

Reply:
Upon further reflection and re-reading https://github.com/httpie/httpie/pull/929#discussion_r439017110 its probably because Requests represents headers in a dict instead of a tuple list. Headers with same name would be lost.

Reply:
Upon further experimentation it turns out its possible to not make use of protected _original for purposes of get_expired_cookies(). Here is an passing implementation https://github.com/IlyaSukhanov/httpie/commit/36daab668c8357d61dd27a954ab8aae556eca202. Can add this commit to this PR if desired.

Reply:
The last iteration on your branch seems reasonable :+1: 
Could you merge it there please?

Reply:
Done